### PR TITLE
Restructure to use redoc.

### PR DESCRIPTION
**Pull request by tbedford**
## Description

* A reorganization of the open specs repository. 
* Also adds in the Control API spec.

- [x] Provides ability to render OAS3 specifications locally using Redoc.
- [x] Updates Control API specification to includes descriptions.
- [x] Fixes issues with the autogenerated Control API spec not being fully OAS3 compliant.
- [x] Adds instructions on importing spec into Postman and trying out the APIs.
- [x] Removes queue endpoint.
- [x] Removes amqp from rules.
- [x] Fixes issue with not being able to specify `"format": "json"`. 
- [x] Adds /me endpoint
- [x] All descriptions added 
- [x] Adds queue endpoint back in.
- [x] All example values added (most have been added)
- [x] Recent changes to rules and queues reflected 
- [x] Reflects latest version of Rules changes based on https://github.com/ably/website/blob/new-rules-format/swagger/v1/swagger.yaml 

## Review

Note to reviewers:

This is currently not perfect, but I think it is in a reasonable state for a Beta release of Control API. I would like to get this merged if possible for the upcoming customer release. We can always raise additional PRs to fix any issues in the spec. Merging allows me to link more easily to this spec from the main docs.

### CODE REVIEW

**Comment by tbedford** [comment]
@mattheworiordan @ginotoro @kavalerov @twist900 - this PR is just so you can test something out locally, for discussion.

File: README.md [hunk 1]
```diff
@@ -1,18 +1,10 @@
...
-Ably provides official client library SDKs developed by Ably that offer a consistent yet idiomatic API across every language. Ably recommends that customers use our comprehensive native SDKs to consume our REST APIs. Download an SDK at [ably.com/download](https://ably.com/download).
-
-If you wish to access the raw REST API however, Ably does provide an OpenAPI and Swagger compatible specification in this repository in [swagger.yaml](./swagger.yaml).
-
-## Viewing the specification
-
-The specification is published online at [https://api-docs.ably.com/docs/service-api](https://api-docs.ably.com/docs/service-api).
-
-In order to view the specification locally, run `nmp install` and `npm run start`, and open the page [localhost:3000/api-docs](http://localhost:3000/api-docs) in your browser.
+In order to view the specification locally, run `http-server -c-1`, and point your browser at `localhost:8080`.
```
Comment (mattheworiordan) [inline README.md:7]: Why did you remove the installation instructions, `npm install`?

Comment (mattheworiordan) [reply README.md:7]: After an `npm install` I could not run `http-server -c-1` anyway.  I think the instructions on how to use this need to be updated.  Are you missing the defintion in the package.json for redoc?

**Review by mattheworiordan — CHANGES_REQUESTED** [review]
I couldn't run the server locally, so sadly could not try this out.  Would you mind updating the docs  and I will try again?

File: openapi/control.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Control API</title>
```
Comment (mattheworiordan) [inline openapi/control.html:4]: Given this is an HTML page, and rendered directly, I think this should mention Ably

Comment (tbedford) [reply openapi/control.html]: Fixed.

**Review by marklewin — APPROVED** [review]
I was able to run this locally using the instructions provided and think it's a great start! I didn't investigate every single entry, but I agree that we can raise separate PRs where appropriate rather than hang everything off this rather monolithic one.

I realise that this focuses on the Control API, but there appear to be missing/blank sample responses in the Platform API and I have [raised an issue](https://github.com/ably/open-specs/issues/9) for those.
